hack/generate/override-snapshot.sh

```diff
@@ -0,0 +1,82 @@
+#!/usr/bin/env bash
+
+set -Eeuo pipefail
+
+# shellcheck disable=SC1091,SC1090
+source "$(dirname "${BASH_SOURCE[0]}")/../lib/__sources__.bash"
+
+function add_component {
+  local component image_ref parameters git_repo revision dockerfile
+  component=${2}
+  image_ref=${3}
+
+  parameters="$(cosign download attestation "${image_ref}" | jq -r '.payload' | base64 -d | jq -c '.predicate.invocation.parameters')"
```

Comment:
not sure if skopeo also supports something like this (I didn't see something so far)